Source/AliveLibAO/ResourceManager.hpp

```diff
@@ -4,6 +4,7 @@
 #include "FunctionFwd.hpp"
 #include "Psx.hpp"
 #include "AnimResources.hpp"
+#include "Factory.hpp"
```

Comment:
this should probably go in ResourceManager.cpp instead then forward declare the enum using `enum class LoadMode : __int16;` inside of  `namespace AO { `